lib/ddtrace/contrib/sidekiq/server_internal_tracer/heartbeat.rb

```diff
@@ -0,0 +1,24 @@
+# typed: true
+
+module Datadog
+  module Contrib
+    module Sidekiq
+      module ServerInternalTracer
+        # Trace Sidekiq server process heartbeats
+        module Heartbeat
+          private
+
+          # This method takes arguments in Sidekiq < 4.2.7
+          def ❤(*) # rubocop:disable Naming/AsciiIdentifiers, Naming/MethodName
+            Datadog.configuration[:sidekiq][:tracer].trace(Ext::SPAN_HEARTBEAT) do |span|
+              span.service = Datadog.configuration[:sidekiq][:service_name]
+              span.span_type = Datadog::Ext::AppTypes::WORKER
+
```

Comment:
You'll need to add the `analytics_enabled` tags in the three new spans created in this PR:
https://github.com/DataDog/dd-trace-rb/blob/80af92a50cf2ad9901109604c04591cdc5009add/lib/ddtrace/contrib/sidekiq/client_tracer.rb#L24-L27

Reply:
Whoops, done!